Makefile

```diff
@@ -4,6 +4,26 @@ NS=texastribune
 DOCKER_ENV_FILE?=env-docker
 LOG_LEVEL?=INFO
 
+# sets up the app to be accessible from localhost
+dev:
+	docker compose up
```

Comment:
<!-- Praises highlight something positive. -->
**praise:** This along with the updated compose file is awesome!